### PR TITLE
Template guide tweaks

### PR DESCRIPTION
**Pull request by hopearmstrong**
Did some cleanup stuff to leave this at a good state.

- Improved the on-boarding flow (added some guidance to the index page and Get Started pages).
- Pulled in the instructions from the GitHub page so that the documentation is visible in the guide.
- Removed the CSS tabs.
- Removed the Download buttons on the Example Templates page (they were non-functional).
- Put in the correct MailChimp unsubscribe merge tags in the templates.
- Probably some other misc. other cleanup things I'm forgetting.

### CODE REVIEW

**Comment by hopearmstrong** [comment]
@anolson Did some last minute tweaks to hopefully wrap this up for the time being. Wanted to make sure I tied up some loose ends before I forgot. I added in the documentation you had on GitHub so it's more clear within the guide that we have those awesome build tools you made.

File: docs/templates/index.html [hunk 1]
```diff
@@ -15,14 +15,14 @@ <h2 class="p-b-1-xs" id="templates">Templates</h2>
                 <p class="p-b-05-xs bold">Basic</p>
                 <p class="p-b-05-xs">ESP Template: none</p>
               </div>
-              <div class="template-action p-tb-1-xs">
+              <!--<div class="template-action p-tb-1-xs">
                 <a href="#" class="regular text-6-xs align-middle-xs inline-xs">
                   <img src="{{ site.baseurl }}/assets/img/icons/download@2x.png" class="template-action-img m-r-1-xs align-middle-xs" alt="Download"> <span class="text-5-xs">Download</span>
                 </a>
-              </div>
+              </div>-->
```
Comment (anolson) [inline docs/templates/index.html]: Let's go ahead and remove these download buttons instead of commented them out.

**Review by anolson — APPROVED** [review]
I left one comment and fixed a few things that HTMLProffer caught.

Otherwise, this looks good to me. 👍

File: docs/index.html [hunk 2]
```diff
@@ -1,21 +1,52 @@
...
+    <p class="p-b-2-xs">Whether you're a marketer, designer, developer, salesperson, or agency, this guide is a resource for building emails at Treehouse. There are a few routes you can take, depending on your goals, needs, and tool requirements.</p>
+    <h3 class="p-b-2-xs">Start by checking out the quick start guides</h3>
+    <a class="m-b-05-xs button button--primary button--large-xs m-r-0-xs" href="{{ site.baseurl}}/getting-started.html" style="margin-top: auto;">Get Started</a>
+    <h3 class="p-t-2-xs">Or take a short cut to find what you're looking for</h3>
+  </div>
+</div>
+<div class="col col-100-xs border-t-light-xs border-b-light-xs p-tb-3-xs block-xs block-md flex-block-lg">
+  <div class="col col-100-xs col-50-md col-25-lg p-lr-2-xs text-center-xs flex-block-xs flex-column-xs">
+    <img src="{{ site.baseurl}}/assets/img/illustrations/principles.png" class="flex-self-baseline-xs w-full-xs p-b-1-xs" alt="Learn Principles">
```
Comment (anolson) [inline docs/index.html:17]: What do you think about this `alt` attribute?

File: docs/molecules/_includes/hero/examples/hero.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
           <!-- // Card Item: Hero -->
           <tr>
             <td class="esg-text-center" align="center">
-              <img src="http://business.teamtreehouse.com/rs/793-KWP-630/images/office-inside1.jpg" width="600" class="esg-round-t esg-image-full">
+              <img src="http://business.teamtreehouse.com/rs/793-KWP-630/images/office-inside1.jpg" width="600" class="esg-round-t esg-image-full" alt="Hero Image">
```
Comment (anolson) [inline docs/molecules/_includes/hero/examples/hero.html:16]: Ditto, about this `alt` attribute.

File: docs/templates/index.html (hunk 1 above)
Comment (hopearmstrong) [reply docs/templates/index.html]: Sounds good, I just removed them and pushed the change.

File: docs/index.html (hunk 2 above)
Comment (hopearmstrong) [reply docs/index.html:17]: Looks good

File: docs/molecules/_includes/hero/examples/hero.html (hunk 3 above)
Comment (hopearmstrong) [reply docs/molecules/_includes/hero/examples/hero.html:16]: Looks good

**Comment by hopearmstrong** [comment]
@anolson Thanks for catching those things. Can you push this to the GitHub page when you get a chance?

**Comment by anolson** [comment]
> Can you push this to the GitHub page when you get a chance?

If you're ready to merge go ahead, GitHub will regenerate the `docs` shortly after it lands on master.

**Comment by hopearmstrong** [comment]
@anolson OH, I didn't know it was automated. Great, will do.

**Pull request merged**